schemas/semconv.schema.json

```diff
@@ -209,7 +209,8 @@
 			"required": [
 				"type",
 				"id",
-				"requirement_level"
+				"requirement_level",
+				"stability"
```

Comment:
I can't come up with the example where the stability of the body type would be different from event stability today. 

In the long term though we will need to be able to define a type and then reuse it across different event bodies and attributes.

E.g. if we have a complex attribute/type for geo location, then we could have `client.geo.location = {"lat": 42, "lon": 42 }` and  `user.geo.location = {"lat": 24, "lon": 24 }` which share the same type, but attribute definitions may have different stability levels.  

So it might look excessive today to provide stability on every step when defining event body, but I don't mind it

Reply:
I agree, it's more consistent this way. If body is not `map` then it's not excessive.